# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../pmud/hotfix.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- pmud-0.10/Makefile.orig	2003-05-15 00:01:04.000000000 +0200
+++ pmud-0.10/Makefile	2003-05-15 00:02:55.000000000 +0200
@@ -52,9 +52,7 @@
 	install -c xmouse /usr/X11R6/bin
 	install -c Batmon /usr/bin
 	install -c -D pwrctl /etc/power/pwrctl
-	cp pmud.rc /etc/rc.d/init.d/pmud
-	ln -s /sbin/snooze /usr/bin/apm
-	chkconfig --add pmud
+	ln -sf /sbin/snooze /usr/bin/apm
 
 clean:
 	rm -f $(PROGS) *.o
